aquacomputer_d5next.c

```diff
@@ -831,7 +840,21 @@ static umode_t aqc_is_visible(const void *data, enum hwmon_sensor_types type, u3
 				switch (attr) {
 				case hwmon_pwm_input:
 				case hwmon_pwm_mode:
```

Comment:
Add `return 0644;` here as the pwm input is independent from the mode.

Reply:
Woops, long day.